### DISCUSSION TITLE
Feature request: various

### DISCUSSION THREAD

**Discussion by ghost** [Steffo99/greed#176] [General]
Many people editing your repo but they are not updating it regularly..  We need those feature at your main repo..

1. Bitcoin Payment system 
2. category, sub category 
3. Instant digital goods sends option
4. custom welcome message 

More clean bot more feature.

**Comment by Steffo99** [comment]
> 1. Bitcoin Payment system

I don't want anything to do with cryptocurrencies.

> 2. category, sub category

You can probably try merging #88 if you need this so much.

> 3. Instant digital goods sends option

I'm not interested in this developing this feature, but I might merge it if I receive a PR of sufficient quality.

> 4. custom welcome message

You can already do that by editing the strings files.

**Reply by ghost** [reply]
Please please add all feature.. Please please please
